packages/extension-ui/src/components/Button.tsx

```diff
@@ -49,9 +49,7 @@ function Button({
     <button
       className={`${className}${isDisabled || isBusy ? ' isDisabled' : ''}${isBusy ? ' isBusy' : ''}${
         secondary ? ' secondary' : ''
-      }${
-        tertiary ? ' tertiary' : ''
-      }`}
+      }${tertiary ? ' tertiary' : ''}`}
```

Comment:
Are those global class names (`tertiary`, `isDisabled` etc.) referenced by any code / css requiring them to be global? If not, I think it's safe to remove these.